survey-runner.tf

```diff
@@ -164,28 +164,128 @@ module "survey-launcher-for-ecs" {
 }
 
 module "author" {
-  source                       = "github.com/ONSdigital/eq-author-deploy"
-  env                          = "${var.env}"
-  aws_access_key               = "${var.aws_access_key}"
-  aws_secret_key               = "${var.aws_secret_key}"
-  vpc_id                       = "${module.survey-runner-vpc.vpc_id}"
-  dns_zone_name                = "${var.dns_zone_name}"
-  ecs_cluster_name             = "${module.eq-ecs.ecs_cluster_name}"
-  aws_alb_arn                  = "${module.eq-ecs.aws_alb_arn}"
-  application_cidrs            = "${concat(var.ecs_application_cidrs, var.application_cidrs)}"
-  docker_registry              = "${var.author_registry}"
-  author_tag                   = "${var.author_tag}"
-  author_api_tag               = "${var.author_api_tag}"
-  publisher_tag                = "${var.publisher_tag}"
-  survey_launcher_url          = "${module.survey-launcher-for-ecs.service_address}"
-  enable_auth                  = "${var.author_enable_auth}"
-  firebase_project_id          = "${var.author_firebase_project_id}"
-  firebase_api_key             = "${var.author_firebase_api_key}"
-  firebase_messaging_sender_id = "${var.author_firebase_messaging_sender_id}"
-  schema_validator_url         = "${module.schema-validator.service_address}"
-  author_min_tasks             = "${var.author_min_tasks}"
-  author_api_min_tasks         = "${var.author_api_min_tasks}"
-  publisher_min_tasks          = "${var.publisher_min_tasks}"
+  source                           = "github.com/ONSdigital/eq-ecs-deploy?ref=346-enable-database-backups-on-pre-production"
+  env                              = "${var.env}"
+  aws_access_key                   = "${var.aws_access_key}"
+  aws_secret_key                   = "${var.aws_secret_key}"
+  dns_zone_name                    = "${var.dns_zone_name}"
+  ecs_cluster_name                 = "${module.eq-ecs.ecs_cluster_name}"
+  vpc_id                           = "${module.survey-runner-vpc.vpc_id}"
+  aws_alb_arn                      = "${module.eq-ecs.aws_alb_arn}"
+  service_name                     = "author"
+  listener_rule_priority           = 102
+  docker_registry                  = "${var.author_registry}"
+  container_name                   = "eq-author"
+  container_port                   = 3000
+  container_tag                    = "${var.author_tag}"
+  healthcheck_path                 = "/status.json"
+  healthcheck_grace_period_seconds = 60
+  slack_alert_sns_arn              = "${module.survey-runner-alerting.slack_alert_sns_arn}"
+
+  application_min_tasks = "${var.author_min_tasks}"
```

Comment:
These are not formatted correctly, we as per we need `terraform fmt`

Reply:
We should also add 
`high_cpu_threshold               = 80`
Because this container builds on launch it breaks the cpu limit and starts scaling.